Sources/SentrySwiftUI/SentryTracedView.swift

```diff
@@ -79,12 +79,21 @@ public struct SentryTracedView<Content: View>: View {
         
         return result
     }
+
+    private func shouldTrace() -> Bool {
+#if DEBUG
+        if  ProcessInfo.processInfo.environment["XCODE_RUNNING_FOR_PREVIEWS"] == "1" {
```

Comment:
If you define that string value as an exported string constant instead of a macro, you should be able to use it here too